Geometry/TrackerNumberingBuilder/interface/GeometricDet.h

```diff
@@ -121,7 +122,7 @@ class GeometricDet {
   double phi() const { return _phi; }
   double rho() const { return _rho; }
 
-  DDSolidShape const& shape() const { return _shape; }
+  cms::DDSolidShape const& shape() const { return _shape; }
```

Comment:
I think, for backward compatibility, `shape()` should return a `LegacySolidShape`. `GeometricDet` is widely used, so it is safest to keep using the old `DDSolidShape` values.
You could add a new version of `shape()` with a new name that returns a ` cms::DDSolidShape`.

Reply:
Ok, I've added a `shape_dd4hep()` to this end